src/deploy/deploy.py

```diff
@@ -262,6 +262,21 @@ def deploy(ctx, release_id, environment_id, description):
     )
 
 
+@cli.command()
+@click.option('--release-id', prompt="Release ID to deploy", default="latest", show_default=True,
+              help="The ID of the release to be deployed, or the latest release if unspecified")
+@click.option('--environment-id', prompt="Environment ID to deploy release to",
+              default="stage", show_default=True,
+              help="The target environment of this deployment")
+@click.pass_context
+def confirm_deploy(ctx, release_id, environment_id):
+    project = ctx.obj['project']
+    release = project.get_release(release_id)
+    ecs_services = project.confirm_service_deployment(release, environment_id)
+    
+    print(ecs_services)
+    return ecs_services
```

Comment:
To use this in CI, we'll need to exit with non-zero if the deployment isn't good within a certain time period. You may need to poll!

Reply:
Added polling and timeout.